spec/defines/openldap_server_access_spec.rb

```diff
@@ -16,7 +16,7 @@
       context 'when Class[openldap::server] is not declared' do
         let(:pre_condition) {}
 
-        it { expect { is_expected.to compile }.to raise_error(%r{class ::openldap::server has not been evaluated}) }
+        it { expect { is_expected.to compile }.to raise_error(%r{class openldap::server has not been evaluated}) }
```

Comment:
This is applicable to multiple files, but just commenting once:
```suggestion
        it { is_expected.to compile.and_raise_error(%r{class openldap::server has not been evaluated}) }
```

Reply:
Done!